support/ebpf/kallsyms.ebpf.c

```diff
@@ -0,0 +1,11 @@
+#include "bpfdefs.h"
+#include "tracemgmt.h"
+#include "types.h"
+
+// kprobe__kallsyms notifies user space about changes to kallsyms.
+SEC("kprobe/kallsysms")
+int kprobe__kallsyms(void *ctx)
+{
+  event_send_trigger(ctx, EVENT_TYPE_RELOAD_KALLSYMS);
```

Comment:
I think this should be rate limited or use the same inhibit mechanism as the other notification.

Reply:
Using the already existing mechanism as the other event with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/1019/commits/6a259a368960b42780a568aceddea94164650e56.